src/matrix/room/sending/PendingEvent.js

```diff
@@ -28,6 +28,10 @@ export const SendStatus = createEnum(
     "Error",
 );
 
+const preservedContentFields = {
+    "m.room.message": [ "m.relates_to" ]
```

Comment:
We can do this independent of the event type I think, just like [element](https://github.com/matrix-org/matrix-js-sdk/blob/a69d9a9f20b680e3710e0f43cca68a2ce31ba030/src/crypto/index.ts#L2753) does. Also, to be correct, we should remove it from the content to be encrypted.